### PR TITLE
JBPM-9445: Stunner - BPMN Showcases refactorings

### PR DESCRIPTION
**Pull request by romartin**
Hey @hasys 

I'd been pushing for this clean up of our code for a while and I think now it's the right moment for it. Also will help on the kogito "fork" we're planning. Also it helps me as I'm rewritting some kogito editor's stuff now and all the showcase-standalone stuff is just making noise...

This ticket/PR does not affect any release, so I think no need for QE reviews, anyway created a tracker ticket for this work: https://issues.redhat.com/browse/JBPM-9445

Please lemme know if it looks good for you and let's continue with this kind of code cleans, which are really necessary!

Thanks!

### CODE REVIEW

**Comment by romartin** [comment]
Jenkins do fdb

**Comment by romartin** [comment]
@hasys WDYT about this PR? I'm refactoring editor classes and woudl be nice to drop some (from the standalone showcase) as soon as possible to make my life easier.. :) Thanks!

File: kie-wb-common-stunner/README.md [hunk 1]
```diff
@@ -23,9 +23,7 @@ Execution and demonstration of the various Stunner-based components can be achie
...
-
-  - [**Stunner Project Showcase**](./kie-wb-common-stunner-showcase/kie-wb-common-stunner-showcase-project/README.md) is built atop the KIE Workbench, Commons, Guvnor and Library components. This more complex showcase demonstrates integration with different KIE Workbench assets and editors. However, for day-to-day development, it is strongly suggested that the Standalone Showcase be utilized over this more complex showcase when working with Stunner components.
+  - [**Stunner BPMN Business Central Showcase**](./kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-showcase/README.md) is built atop the KIE Workbench, Commons, Guvnor and Library components. This more complex showcase demonstrates integration with different KIE Workbench assets and editors. However, for day-to-day development, it is strongly suggested that the Standalone Showcase be utilized over this more complex showcase when working with Stunner components.
```
Comment (hasys) [inline kie-wb-common-stunner/README.md:26]:
```suggestion
  - [**Stunner BPMN Business Central Showcase**](./kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-project-showcase/README.md) is built atop the KIE Workbench, Commons, Guvnor and Library components. This more complex showcase demonstrates integration with different KIE Workbench assets and editors. However, for day-to-day development, it is strongly suggested that the Kogito Standalone Showcase be utilized over this more complex showcase when working with Stunner components.
```

**Comment by hasys** [comment]
Hi @romartin, code changes looks good to me, but I can't log in into the project now.
image

I tried `admin/admin` and my Wildfly users/roles but still without success. So potentially not able to test my changes. Can we remove this authentication at all? I don't think it is needed there.

Otherwise great job! I am agree it is good to simplify the code base.

File: kie-wb-common-stunner/README.md (hunk 1 above)
Comment (romartin) [reply kie-wb-common-stunner/README.md]: thanks  @hasys , change applied!

**Comment by romartin** [comment]
Hey @hasys 

I just tested again the project-showcase and I can properly login 
- in SDM ->  by using `admin/admin` when by command line (`mvn clean gwt:run`)
- in Wildfly -> also by deploying the war generated into a wildfly instance, using the login configured in that local instance

Maybe can you clean browsers' cache and reload, and test again? 

Thanks!

**Comment by romartin** [comment]
jenkins do fdb

**Comment by hasys** [comment]
Hi @romartin,

`mvn gwt:run` works good to me, thank you!

**Review by hasys — APPROVED** [review]

**Pull request merged**